brain-marks/Settings/SettingsViewModel.swift

```diff
@@ -54,4 +54,28 @@ class SettingsViewModel: ObservableObject {
         }
         .resume()
     }
+
+  func getStringFromBundle() -> String {
```

Comment:
I would rename this to be more specific that this is only for getting the `whatsnew` file, maybe something like `getWhatsNewFromBundle()`??